### PR TITLE
Add a contact form

### PR DESCRIPTION
**Pull request by l-vincent-l**

### CODE REVIEW

**Comment by l-vincent-l** [comment]
The component is not revelant anymore

**Comment by maukoquiroga** [comment]
# /contact_form

capture d ecran 2017-11-22 a 14 29 51

**Comment by maukoquiroga** [comment]
It still needs :

- [ ] A test (solution?)
- [ ] Rebuild po files

File: priv/gettext/fr/LC_MESSAGES/contact.po [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## (with the same domain).
+##
+## Use `mix gettext.extract --merge` or `mix gettext.merge`
+## to merge POT files into PO files.
+msgid ""
+msgstr ""
+"Language: fr\n"
+
+#: lib/transport_web/controllers/contact_controller.ex:13
+msgid "Message sent, we will contact you soon."
```
Comment (maukoquiroga) [inline priv/gettext/fr/LC_MESSAGES/contact.po]: This should be french...

Comment (maukoquiroga) [reply priv/gettext/fr/LC_MESSAGES/contact.po]: Bump.

**Comment by maukoquiroga** [comment]
Closing as stale. Reopen otherwise.

**Pull request closed without merging**